instagram-app/src/Components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import CommentSection from '../CommentSection/CommentSection.js';
+import heart from "./heart-regular.svg"
+import './PostContainer.css';
+
+// const PostContainer = props => {
+//   console.log(props);
+//   return (
+//     <div className="postcontainer">
+//       <a href="#">
+//         <img src={props.eachPost.thumbnailUrl}/>
+//         {props.eachPost.username}
+//       </a><br/>
+//       <img src={props.eachPost.imageUrl} />
+//       <br/>
+//       <img src={heart} onClick={props.likeCounter} width="30px"/>
+//       <p>{`${props.eachPost.likes} Likes`}</p>
+//       <CommentSection comments={props.eachPost.comments}/>
+//     </div>
+//   );
+// }
+
+class PostContainer extends React.Component {
+  constructor(props){
+    super(props);
+    this.state={
+      ...props.eachPost,
+    }
+    console.log(this.state);
+  };
+

```

Comment:
This click handler brings up a warning message saying. "Do not mutate state directly. Use setState()." Basically it doesn't like the use of *this.state.likes* inside the setState(). A quick fix is to assign a variable to that increments the likes.
`let likes = this.state.likes += 1;
    this.setState({
      likes: likes,
    })`
or using ES6
`let likes = this.state.likes += 1;
    this.setState({ likes })`